PubNubUnity/Assets/PubNub/Workers/SubscriptionWorker.cs

```diff
@@ -334,7 +334,7 @@ private void RunSubscribeRequest (long timetoken, bool reconnect)
                 return;
             }
 
-            if(!string.IsNullOrEmpty(this.PubNubInstance.AuthorizedUUID) && !this.PubNubInstance.AuthorizedUUID.Equals(this.PubNubInstance.PNConfig.UserId)){
+            if(!string.IsNullOrEmpty(this.PubNubInstance.AuthorizedUserId) && !this.PubNubInstance.AuthorizedUserId.Equals(this.PubNubInstance.PNConfig.UserId)){
                 string responseMsg = "Authorized UUID doesn't match.";
```

Comment:
UUID in "Authorized UUID doesn't match."?

Reply:
Fixed the message.